indico_themes_canonical/static/css/ubuntu-summit/header.css

```
@@ -0,0 +1,68 @@
+/* ----------- Styles for the conference header ----------- */
+
+/*
+Wraps around the upper part of the conference header.
+*/
+.confheader {
+  background-blend-mode: multiply, multiply, normal, normal, normal;
+  background-image: linear-gradient(
+      to bottom right,
+      rgb(228 228 228 / 54%) 0%,
+      rgb(228 228 228 / 54%) 49.9%,
+      rgb(228 228 228 / 0%) 50%,
+      rgb(228 228 228 / 0%) 100%
+    ),
+    linear-gradient(
+      to bottom left,
+      rgb(216 216 216 / 54%) 0%,
+      rgb(216 216 216 / 54%) 49.9%,
+      rgb(216 216 216 / 0%) 50%,
+      rgb(216 216 216 / 0%) 100%
+    ),
+    linear-gradient(
+      to top right,
+      rgb(255 255 255 / 100%) 0%,
+      rgb(255 255 255 / 100%) 49%,
+      rgb(255 255 255 / 0%) 50%,
+      rgb(255 255 255 / 0%) 100%
+    ),
+    linear-gradient(
+      to top right,
+      rgb(255 255 255 / 100%) 0%,
+      rgb(255 255 255 / 100%) 100%
+    ),
+    linear-gradient(-89deg, #e47a54 0%, #bb4115 38%, #772507 85%);
+  background-position: 0% 0%, top right, right 0 bottom 4rem, right bottom,
+    0% 0%;
+  background-repeat: no-repeat;
+  background-size: 100% calc(100% - 4rem), 50% 100%, 100% 4rem, 100% 4rem, auto;
+  margin-bottom: -4rem;
+  overflow: hidden;
+  padding-bottom: 12rem !important;
```

The header strip needs the same width and left/right padding as the `#confSectionsBox` below it - if we can't use SCSS, and since we're not supporting IE11, you could use CSS custom properties to set common values like this for the whole project (https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties):
```suggestion
  padding-bottom: 12rem !important;
  padding-left: 20px;
  padding-right: 20px;
  width: 1242px;
```

---

I have created the variable and applied them to the appropriate classes. However, I am not clear where I need to apply the width variable for the header? I am going to leave it for now as this was only a placeholder header until the brand issue is complete with the new header design.